### DISCUSSION TITLE
pillar binning

### DISCUSSION THREAD

**Discussion by konyul** [pyg-team/pytorch_geometric#6408] [General]
Hi, I am struggling with some problem
the problem is to convert a pillar to several voxel grids
A pillar has a shape of (20000,20,5). 20000 is the number of pillars,  20 is the number of points inside the pillar and 5 is the coordinates of each point in a pillar

the shape i want these pillars to be converted is (20000,32,5). 20000 is the number of pillars, 32 is the grid divided inside a pillar which is predefined. and 5 is the average coordinates of the points which is inside that grid divided inside a pillar

Is there any function that would solve this problem??

**Comment by rusty1s** [comment]
So you want to map each point to its closest voxel cell? There exists a `voxel_grid` function in PyG which may be able to achieve this.
